models/rover_ackermann/model.sdf

```diff
@@ -336,7 +336,7 @@
 	      <limit>
 		<lower>-1.79769e+308</lower>
 		<upper>1.79769e+308</upper>
```

Comment:
Could you check if these upper and lower limits can be removed for the new Gazebo? These were added as part of a workaround with some issues when enabling lockstep with Gazebo Classic

Reply:
It actually works without them, removed them in [a6c7ef4](https://github.com/PX4/PX4-gazebo-models/pull/67/commits/a6c7ef4bb5f27aa35a649e2a14a016c43ff7e3ac)